config/schema-registry.properties

```diff
@@ -42,3 +42,9 @@ kafkastore.topic=_schemas
 
 # If true, API requests that fail will include extra debugging information, including stack traces
 debug=false
+
+# If true, try to write commit logs for backup up schemas topic
+schema.registry.write.backups=true
```

Comment:
should be false by default

Reply:
made this change - however, to confirm, this is just for local runs, do we still want this to be the case?